### PR TITLE
test are passing + tags 

### PR DESCRIPTION
**Pull request by chantal66**

### CODE REVIEW

File: spec/features/companies/user_can_see_office_locations_assocaited_w_company_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "User can see office locations on a specific company" do
+  scenario "User can see office locations on a specific company" do
+    company = Company.create!(name: "ESPN")
+    location = company.locations.create!(location: 'Denver')
+
```
Comment (laurenfazah) [inline spec/features/companies/user_can_see_office_locations_assocaited_w_company_spec.rb]: great setup

File: app/controllers/companies_controller.rb [hunk 2]
```diff
@@ -18,8 +18,8 @@ def create
   end
 
   def show
-    company = Company.find(params[:id])
-    redirect_to company_jobs_path(company)
+    @company = Company.find(params[:id])
+    # redirect_to company_jobs_path(company)
   end
```
Comment (laurenfazah) [inline app/controllers/companies_controller.rb]: Just be careful of what kind of side effects changing source code could have...

File: app/models/company.rb [hunk 3]
```diff
@@ -4,4 +4,8 @@ class Company < ActiveRecord::Base
   has_many :jobs
   has_many :officings
   has_many :locations, through: :officings
+
+  def count(company, location)
+    company.locations.where(location: location).count
+  end
```
Comment (laurenfazah) [inline app/models/company.rb]: Do we need to pass `company` into its own model?

File: app/controllers/companies_controller.rb (hunk 2 above)
Comment (chantal66) [inline app/controllers/companies_controller.rb]: this part right here was a little tricky... I couldn't think in a better solution at that time... I know was breaking other test. I'm sorry

**Comment by laurenfazah** [comment]
Much better this time around! My biggest concern was your change of code that existed in the codebase before you implemented your solution. Be careful doing this - in larger projects with teams, this may have negative side effects elsewhere throughout the codebase. Otherwise, well done.

**Pull request closed without merging**